mlflow/pipelines/helpers.py

```diff
@@ -0,0 +1,111 @@
+from abc import ABC, abstractmethod
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+from typing import Optional
+
+
+class PipelineHelper(ABC):
```

Comment:
I think this approach looks great currently. One thing that I am worried is that as we build out more templates and more complex templates this `PipelineHelper` class may contain arbitrary functions without a cohesive story.

Is this a problem? And if so, how would we solve it?